wandb/sdk/lib/server.py

```diff
@@ -36,3 +37,20 @@ def query_with_timeout(self, timeout: int | float = 5) -> None:
         # TODO(jhr): should we kill the thread?
         self._viewer, self._serverinfo = viewer_tuple
         self._flags = json.loads(self._viewer.get("flags", "{}"))
+
+    @property
+    def viewer(self) -> dict[str, Any]:
+        """
+        Returns the viewer information queried from the backend `viewer` GraphQL API.
+
+        The viewer contains information about the currently authenticated user, including:
+        see the GraphQL viewer query defined here:
+        https://github.com/wandb/wandb/blob/main/wandb/sdk/internal/internal_api.py#L948-L966
```

Comment:
this is not a permanent link, so i don't think it is a good idea to have it here if we move the files this would be busted.